hop-{{cookiecutter.app_name}}-app/hop/apps/{{cookiecutter.app_name}}/__main__.py

```diff
@@ -7,10 +7,10 @@
 
 
 def _set_up_parser():
-    """Set up parser for scimma app entry point.
+    """Set up parser for {{ cookiecutter.app_name }} app entry point.
 
     """
-    parser = argparse.ArgumentParser(prog="scimma-{{ cookiecutter.app_name }}")
+    parser = argparse.ArgumentParser(prog="{{ cookiecutter.app_name }}")
```

Comment:
I would switch `scimma` to `hop` instead, to match the entry point defined in https://github.com/scimma/hop-app-template/blob/master/hop-%7B%7Bcookiecutter.app_name%7D%7D-app/setup.py#L32

Reply:
okay will change that